target/snitch_cluster/Makefile

```diff
@@ -82,6 +85,7 @@ VLOG_FLAGS += -suppress 13314
 VLOG_FLAGS += ${VLOG_64BIT}
 
 VLT_FLAGS  += --trace
+VLT_FLAGS  += --trace-structs
```

Comment:
With this, we can see the hierarchy in vcd file which is convenient for debug

Reply:
I'm referencing the documentation here at https://veripool.org/guide/latest/exe_verilator.html#cmdoption-trace-structs.
 
> Enable tracing to show the name of packed structure, union, and packed array fields, rather than a single combined packed bus. Due to VCD file format constraints, this may result in significantly slower trace times and larger trace files.

Enabling this in CI might slow down things unnecessarily! I'd recommend to keep this somewhere if you'd like to use this personally, but we should remove this in our git repo.